### PR TITLE
Adding ReRAM PDK to the submodules directory of the skywater repo

### PR DESCRIPTION
**Pull request by QuantamHD**
This pull request adds ReRAM to the Skywater PDK. ReRAM is implemented a MiM type device between metal1 and metal2 in the layer stack up. These device with special programming allow you to store up to two bits of information per device; taking up the same amount of space as a metal1 to metal2 via. 

Signed-off-by: Ethan Mahintorabi <ethanmoon@google.com>

Fixes google/skywater-pdk-libs-sky130_fd_pr_reram#7 

- [ x ] Tests pass
- [ x ] Appropriate changes to README are included in PR

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -244,3 +243,20 @@
 	branch = branch-0.0.1
 	shallow = true
 	fetchRecurseSubmodules = false
+#sky130_fd_pr_reram
+[submodule "libraries/sky130_fd_pr_reram/v2.0.1"]
+	path = libraries/sky130_fd_pr_reram/v2.0.1
+	url = https://github.com/google/skywater-pdk-libs-sky130_fd_pr_reram.git
+	branch = branch-2.0.1
```
Comment (mithro) [inline .gitmodules:250]: Please add the extra properties found under the other submodule definitions.

**Comment by mithro** [comment]
Please also fix up the pull request's description.

**Comment by QuantamHD** [comment]
@mithro Can you give it one last look, and merge if you think it looks good.

File: .gitmodules [hunk 2]
```diff
@@ -244,3 +243,28 @@
 	branch = branch-0.0.1
 	shallow = true
 	fetchRecurseSubmodules = false
+#sky130_fd_pr_reram
```
Comment (mithro) [inline .gitmodules:246]: Your missing the `latest` submodule;
```
[submodule "libraries/sky130_fd_io/latest"]
	path = libraries/sky130_fd_io/latest
	url = https://github.com/google/skywater-pdk-libs-sky130_fd_io.git
	branch = main
	shallow = true
	fetchRecurseSubmodules = true
```

Can you make sure the new modules are also sorted correctly? (newest -> oldest)

File: .gitmodules [hunk 3]
```diff
@@ -244,3 +243,28 @@
 	branch = branch-0.0.1
 	shallow = true
 	fetchRecurseSubmodules = false
```
Comment (mithro) [inline .gitmodules:245]: Newline

**Pull request merged**